src/cljs/rems/handlers.cljs

```diff
@@ -36,3 +37,18 @@
  :set-current-language
  (fn [db [_ language]]
    (assoc db :language language)))
+
+(reg-event-fx
```

Comment:
qould it be clearer if this was just a function (in spa.cljs), after all it has all sorts of side effects like `dispatch!` and `removeItem`

Reply:
I think we should dismantle the handlers.cljs altogether. Perhaps now is the time? Same for subs?

Reply:
fine by me :)